src/style.css

```
@@ -1 +1,96 @@
-/* Styles go here */
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+}
+
+li {
+  list-style: none;
+}
+
+* {
+  all: unset;
+  display: revert;
+  margin: 0;
+  box-sizing: border-box;
```

you can apply only box-sizing: border-box; for `*` selector
```suggestion
  box-sizing: border-box;
```